packages/ui/src/ui/components/StatisticTable/prepare-job-statistic.ts

```diff
@@ -0,0 +1,29 @@
+import hammer from '../../common/hammer';
+import {JobStatistic, JobTree, MetricsEntry} from '../../types/job';
```

Comment:
It should use types defined for `StatistcTable.tsx` instead of ones from `../../types/job`